devenv.nix

```diff
@@ -43,6 +43,36 @@ in
     '';
   };
 
+  # convenient shortcuts
+  scripts.up.exec = "devenv up -d";
+  scripts.up.description = "Start processes in the background.";
+
+  scripts.down.exec = "devenv processes down";
+  scripts.down.description = "Stop processes.";
+
+  scripts.tests.exec = "pytest && rm -rf ./tmp";
+  scripts.tests.description = "Run pytest and cleanup.";
```

Comment:
For `tests` it's better to just run `./tests.sh` as it carries all the tests. And can you elaborate on what is `./tmp`? If it's temporary let's put it in `/tmp` instead.

Reply:
@shahinism `./tmp` is a folder that gets automatically created by current tests, look 
<img width="312" alt="image" src="https://github.com/user-attachments/assets/33c25471-aae8-4c83-82a1-1e9b20ee2063">.
It doesn't do anything bad, I just find it distracting to have it there hanging.

As for `./tests.sh`, had missed that, I'll update.

Reply:
<img width="323" alt="image" src="https://github.com/user-attachments/assets/52be780c-92b6-43ce-8adc-96827138d915">
here an example of where that local `./tmp/` comes from.

Reply:
Addressed in f7d5c14